code/script.js

```diff
@@ -1,9 +1,34 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
-
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+//JavaScript for the Accordion
+
+const accordionHeader1 = document.getElementById("accordion-header-1");
+const accordionBody1 = document.getElementById("accordion-body-1");
+
+
+
```

Comment:
We don't need to add so many empty lines - one is completely enough. Let's fix that